src/config.cpp

```diff
@@ -89,19 +89,33 @@ void Config::setupConfig(Json::Value &dst, const std::string &config_file, int d
   mergeConfig(dst, tmp_config);
 }
 
+std::optional<std::string> Config::findIncludePath(const std::string name) {
```

Comment:
Pass by reference to avoid the extra copy.
```suggestion
std::optional<std::string> Config::findIncludePath(const std::string& name) {
```